### ISSUE TITLE
Update Facemesh Description

### ISSUE THREAD

**Issue by B2xx** [ml5js/ml5-website-v02-docsify#41]
Hi everyone! I help to develop the facemesh model for [ml5-next-gen](https://github.com/ml5js/ml5-next-gen) last summer and I plan to update some descriptions for[ facemesh models in ml5 website](https://learn.ml5js.org/#/reference/facemesh). This is my [prototype](https://docs.google.com/document/d/1EIu20MsEYZFcMTykO2W5eSTzKGIKHTDQXqbfnu4EPWs/edit?usp=sharing), and welcome to comment!

This is my [proposal](https://docs.google.com/document/d/162S5ADrMlXOfV3GTNwqK9-u38fDR46ejJHVv0QqBh1c/edit?usp=sharing), and I want to update the content for facemesh in ml5.js website so that it can be consistent with the [Tensorflow.js (tfjs)'s documentation](https://github.com/tensorflow/tfjs-models/tree/master/face-landmarks-detection). I mainly want to update [**Methods and Usage** ](https://learn.ml5js.org/#/reference/facemesh?id=usage) (You need to scroll down a little bit to find the Usage part) sections for the website as the model changes greatly. 

Besides, I also notice some inconsistencies in the descriptions for each model. I'm wondering if we could make a template to describe each model.  I think adding a sentence introducing its possible application would be helpful. 

I mentioned in the last part of the proposal that I'd also to implement blank **examples, demos, and tutorials** parts. I'm wondering if anyone finds any great resources like [this snippet](https://github.com/apple2373/mediapipe-facemesh/tree/main) to create a .obj file from the facemesh. Or is there anything specifically you want to learn with the facemesh, or anything inspiring with general facial recognition technology?

Thank you!

**Comment by MOQN** [comment]
@B2xx This is amazing! Can you please revise the markdown accordingly and make a PR? We will review and apply the updates.

**Comment by B2xx** [comment]
> @B2xx This is amazing! Can you please revise the markdown accordingly and make a PR? We will review and apply the updates.

Hi Moon, it's nice to hear from you! I'd like to make a PR but since I haven't done any, could you give me an example/tutorial?  Thank you!

**Comment by MOQN** [comment]
Hi @B2xx ! Please reference [the slides](https://docs.google.com/presentation/d/1dkfgyQNVizyFORzf4fYIodz1n6I1iQi_G6p3nsQ-ijw/edit?usp=sharing) that shared in the class! You may look at the slides from Slide #65. I will follow up after the midterm here is done!

**Comment by shiffman** [comment]
Hi @B2xx thank you for sharing your work! While accessibility is relevant to all issues and discussions, we are reserving the `a11y` label for specific issues related to WCAG compliance. This is a more general discussion of documentation.

If you would like help with making your pull request, feel free to sign up for office hours!

**Comment by QuinnHe** [comment]
Hi @B2xx ! Nice work! For your reference, this is the latest version of [Facemesh model page](https://ml5js.github.io/ml5-website-v02-docsify/#/reference/facemesh), [the current markdown of Facemesh](https://github.com/ml5js/ml5-website-v02-docsify/blob/main/docs/reference/facemesh.md) should be a good place for you to incorporate your changes and create a PR.

**Comment by B2xx** [comment]
Thank you! If I understand correctly, I will need to create a fork and started a pull request there, right?

**Comment by QuinnHe** [comment]
> Thank you! If I understand correctly, I will need to create a fork and started a pull request there, right?

Yes, just remember to select "To contribute to the parent project." just as slide 69 showed. That way, we will able to review your changes on our end :)

**Comment by B2xx** [comment]
> > Thank you! If I understand correctly, I will need to create a fork and started a pull request there, right?
> 
> Yes, just remember to select "To contribute to the parent project." just as slide 69 showed. That way, we will able to review your changes on our end :)

Sure, thank you! I'm sorry that I've been a little busy on Spring break and last week but will update this as soon as possible.
